### DISCUSSION TITLE
Welcome to pythermalcomfort Discussions!

### DISCUSSION THREAD

**Discussion by FedericoTartarini** [CenterForTheBuiltEnvironment/pythermalcomfort#43] [Announcements]
# Welcome to pythermalcomfort Discussions! 👋

Thank you for your interest in pythermalcomfort! Before posting, please take a moment to understand what this space is designed for.

## What are Discussions for?

GitHub Discussions is a community forum where we can:

- **Ask questions** about how to use pythermalcomfort
- **Share ideas** and general thoughts about thermal comfort research
- **Start conversations** about best practices and methodologies
- **Show off projects** you've built using pythermalcomfort
- **Get help** with implementation challenges or usage questions
- **Discuss** thermal comfort standards, research, or industry topics

## ⚠️ Important: This is NOT the place for bug reports or feature requests

If you're experiencing a problem with the library or want to request a new feature, please use the appropriate section:

### 🐛 Found a Bug?
**Use the [Issues](https://github.com/CenterForTheBuiltEnvironment/pythermalcomfort/issues) section instead** - Click the "Issues" button and create a new issue by selecting the appropriate template. Always remember to include:
- A clear description of the problem
- Steps to reproduce the issue
- Your environment details (Python version, pythermalcomfort version, etc.)
- Error messages or unexpected behavior

### 💡 Want a New Feature?
**Use this [link](https://github.com/CenterForTheBuiltEnvironment/pythermalcomfort/issues/new/choose)** - From the templates choose 'feature' and please include:
- A clear description of the desired functionality
- Use cases explaining why this feature would be valuable
- Any relevant references or standards

## Getting the Most Out of Discussions

When posting a discussion:
- Use a descriptive title
- Provide context about what you're trying to achieve
- Share relevant code snippets if asking for implementation help
- Be respectful and constructive in your interactions

**Comment by Fedovith** [comment]


**Reply by FedericoTartarini** [reply]
@Fedovith, thank you for using pythermalcomfort. I have converted your discussion into an issue, let's continue the conversation [there](https://github.com/CenterForTheBuiltEnvironment/pythermalcomfort/issues/218). In the meantime I will delete this discussion.
